README.md

```diff
@@ -24,9 +24,10 @@ This dependency contains the GraalVM configurations to provide out-of-the-box su
 
 To compile with GraalVM (native-image), ensure the client library version in your project is supported by `google-cloud-graalvm-support`.
 
-| GraalVM Support version | *`libraries-bom` version | `grpc-netty-shaded` version |
-|-------------------------|:-------------------------|-----------------------------|
-| `0.4.0`                 | `11.0.0` or later        | `1.32.1` or later           |
+| GraalVM Support version | GraalVM Compiler Version | *`libraries-bom` version | `grpc-netty-shaded` version |
+|-------------------------|--------------------------|:-------------------------|-----------------------------|
+| `0.4.0`                 | `>= 21.0.0`              | `11.0.0` or later        | `1.32.1` or later           |
+| `0.5.0`                 | `>= 21.2.0`              | `20.8.0` or later        | `1.39.0` or later           |
```

Comment:
I feel it's natural to have newer entries come first.

Reply:
Done! Agreed, makes sense.